src/control/cmd/daos/container.go

```diff
@@ -1444,3 +1445,32 @@ func (cmd *containerLinkCmd) Execute(_ []string) (err error) {
 
 	return nil
 }
+
+type containerEvictCmd struct {
+	existingContainerCmd
+
+	EvictAll bool `long:"evict-all" short:"a" description:"evict all handles from all users"`
```

Comment:
Why `--evict-all` instead of `--all`?

Reply:
yea i was debating that while adding this and thought --all was too short or ambiguous but was repetitive.  if it sounds better i'll make the change

Reply:
ill update this in a follow on PR